frontend/src/metabase/browse/components/utils.tsx

```diff
@@ -70,6 +80,37 @@ export const sortModels = (
   });
 };
 
+export const sortMetrics = (
```

Comment:
This is the same as `sortModels`. We can rename `sortModels` to `sortCards` and make it accept `MetricResult[]` as well.